docs/rule/attribute-indentation.md

```diff
@@ -79,6 +103,19 @@ Block form (open invocation < 80 characters)
   {{/employee-details}}
 ```
 
+Block form (HTML)
+``` html
+
+  <a
+    href="https://www.emberjs.com"
+    class="emberjs-home link"
+    rel="noopener"
+    target="_blank">
+    Ember JS
+  </a>
+```
+
 #### Configuration:
   * boolean - `true` - Enables the rule to be enforced when the opening invocation has more than 80 characters or when it spans multiple lines.
```

Comment:
Can you update this to show exactly what `true` expands to (e.g. I assume it does not enable `process-elements`, but we should be explicit)

Reply:
Done, also updated the error message in `parseConfig` to specify that when `true` it will only apply to handlebar components and not HTML/SVG elements.